docs/Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'github-pages'
-
-gem "webrick", "~> 1.7"
+
```

Comment:
Can we revert this file? Looks like nothing has changed.

Reply:
Done